Makefile

```diff
@@ -46,4 +48,12 @@ release:
 	opam publish submit $(NAME_VERSION)
 	rm -rf $(NAME_VERSION)
 
+install:
+	ocamlfind remove ppx_deriving
+	grep -E '^[[:space:]]+' ppx_deriving.install | \
```

Comment:
The goal was to avoid information duplication on the file to copy, do you have another idea in mind?